modules/local/payload/resources/usr/bin/main.py

```diff
@@ -73,13 +73,13 @@ def rename_file(f, payload, seq_experiment_analysis_dict, date_str):
     else:
         sys.exit('Error: unknown variant type: %s' % f)
 
-    new_name = "%s.%s.%s.%s.%s.%s.%s" % (
+    new_name = "%s.%s.%s.%s.%s.somatic.germline.%s.%s" % (
```

Comment:
Use a different separator, `.` is already used for the various metadata fields.

Having `somatic.germline` would confuse someone parsing the file name. Perhaps `somatic-germline` instead